docs/_docs/monitoring-metrics/performance-statistics.adoc

```diff
@@ -0,0 +1,85 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+= Performance statistics
+
+WARNING: This feature is experimental and can be changed in future releases.
+
+== Overview
+
+Ignite provides a built-in tool for cluster profiling.
+
+You can link:#collecting-statistics[collect] performance statistics from the cluster and then link:#building-the-report[build] the performance report.
+
+== Collecting statistics
+
+To start and stop collecting statistics use the link:#jmx[JMX interface].
+
+Each node collects performance statistics to the file in binary format. This file is placed under `Ignite_work_directory/perf_stat/` directory.
+The name mask is `node-{nodeId}-{index}.prf`.
+
+Performance statistics files are used to build the report offline. Stop collecting statistics before creating the report.
+
+Nodes use off-heap buffer to temporarily store serialized statistics. The writer thread flushes buffer to the file when the flush size is reached. See the link:#system-properties[properties] section for details.
+
+Each start of collecting statistics will create a new file on nodes. The index in the file name will be increased if such file exists:
```

Comment:
Each statistics collection process creates a new file on nodes. Each next file has the same name with the corresponding index:

Reply:
Fixed.